src/operator/webhooks/protectedservices_webhook.go

```diff
@@ -76,6 +76,15 @@ func (v *ProtectedServiceValidator) ValidateUpdate(ctx context.Context, oldObj,
 	var allErrs field.ErrorList
 	protectedServices := newObj.(*otterizev1alpha2.ProtectedService)
 
+	protectedServicesList := &otterizev1alpha2.ProtectedServiceList{}
+	if err := v.List(ctx, protectedServicesList, &client.ListOptions{Namespace: protectedServices.Namespace}); err != nil {
+		return err
+	}
+
+	if err := v.validateNoDuplicateClients(protectedServices, protectedServicesList); err != nil {
```

Comment:
Need to call this on `ValidateCreate` as well

Reply:
whoops